mne/epochs.py

```diff
@@ -137,11 +137,12 @@ def __init__(self, info, event_id, tmin, tmax, baseline=(None, 0),
         if decim > 1:
             new_sfreq = sfreq / decim
             lowpass = self.info['lowpass']
-            if new_sfreq < 2.5 * lowpass:  # nyquist says 2 but 2.5 is safer
+            if (lowpass is not None) and (new_sfreq < 2.5 * lowpass):
```

Comment:
to me if lowpass is None then no filter has been applied to data is likely to be unfiltered and the warning should happen

wdyt?